adminlogin.aspx.cs

```diff
@@ -9,9 +11,45 @@ namespace LogBook
 {
     public partial class adminlogin : System.Web.UI.Page
     {
+        string strcon = ConfigurationManager.ConnectionStrings["con"].ConnectionString;
+
         protected void Page_Load(object sender, EventArgs e)
         {
 
         }
+
+        protected void Button1_Click(object sender, EventArgs e)
+        {
+            try
+            {
+                SqlConnection con = new SqlConnection(strcon);
+                if (con.State == System.Data.ConnectionState.Closed)
+                {
+                    con.Open();
+                }
+                SqlCommand cmd = new SqlCommand("select * from admin_login_tbl where username='" + TextBox1.Text.Trim() + "' AND password='" + TextBox2.Text.Trim() + "'", con);
+                SqlDataReader dr = cmd.ExecuteReader();
+                if (dr.HasRows)
+                {
+                    while (dr.Read())
+                    {
+                        //Response.Write("<script>alert('" + dr.GetValue(0).ToString() + "');</script>");
+                        Session["username"] = dr.GetValue(0).ToString();
+                        Session["fullname"] = dr.GetValue(2).ToString();
+                        Session["role"] = "admin";
```

Comment:
### Set _constant_ values instead

Use constant values instead of user defined values in these lines of code. 
```suggestion
                       // declare this all the way at the top before the Page_Load() function
                        const string UserTypeAdmin = "admin";

                       // when you need to use the variable, just pass it in. 
                       Session["role"] = UserTypeAdmin;
```